test/index.js

```diff
@@ -15,24 +16,26 @@ describe('inquirer-npm-name', function () {
     this.inquirer = {
       prompt: sinon.stub()
     };
+
   });
 
   it('only ask name if name is free', function (done) {
-    this.inquirer.prompt.callsArgWith(1, {name: 'foo'});
 
-    askName(this.conf, this.inquirer, function (val) {
-      assert.equal(val, 'foo');
+    this.inquirer.prompt.returns(Promise.resolve({name:'foo'}));
+
+    return askName(this.conf, this.inquirer).then(function (props) {
+      assert.equal(props.name, 'foo');
       done();
```

Comment:
If I remove the `done()` call, I catch the error below: 

**_timeout of 2000ms exceeded. Ensure the done() callback is being called in this test_**

I think that we need use the `mocha` npm package that handler the returned **Promise** into a `it()` spec. What do you think?